extensions/wikia/UserLogin/js/UserBaseAjaxForm.js

```diff
@@ -10,8 +10,13 @@
 	 * @constructor
 	 */
 	var UserBaseAjaxForm = function (el, options) {
+		if (!(el instanceof HTMLElement) && !(el instanceof jQuery)) {
+			throw 'This module requires an html element or jQuery object as the first parameter';
```

Comment:
Hey, what do you think about throwing a "new Error('This module...');" ?
This would give more debugging information (stacktrace etc.).